var/spack/repos/builtin/packages/cdo/add_algorithm_header_222.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/src/cdo_options.cc b/src/cdo_options_patched.cc
+index 465f1f9..0684e78 100644
+--- a/src/cdo_options.cc
++++ b/src/cdo_options_patched.cc
+@@ -12,6 +12,7 @@
+ #include "cdo_output.h"
+ 
+ #include <cstring>
++#include <algorithm>
+ 
+ namespace cdo
+ {
+diff -u src/cdo_options.h src/cdo_options_patched.h                                                                            [Mon 2024-09-02|16:00:46]
```

Comment:
```suggestion
diff -u src/cdo_options.h src/cdo_options_patched.h
```

Reply:
and sha256 below.

Reply:
> and sha256 below.

what do you mean?

Reply:
Right, I forgot about this. Thanks for the clarification